src/main/java/se/sundsvall/chat/api/model/Message.java

```diff
@@ -0,0 +1,170 @@
+package se.sundsvall.chat.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Objects;
+
+@Schema(description = "Represents a message within a conversation, including metadata and content.")
+public class Message {
+
+	@Schema(description = "The unique identifier of the message.", example = "7f77f9fd-d01d-4742-974a-714b911e3496")
+	private String id;
+
+	@Schema(description = "The sequence number of the message in the conversation.", example = "1")
+	private String sequenceNumber;
+
+	@Schema(description = "The identifier of the message this message is replying to, if any.", example = "msg-12344")
+	private String inReplyTo;
+
+	@Schema(description = "The timestamp when the message was created.", example = "2023-01-01T12:00:00")
+	private OffsetDateTime created;
+
+	@Schema(description = "The participant who created the message.")
+	private Participant createdBy;
+
+	@Schema(description = "The content of the message.", example = "Hello, how can I help you?")
+	private String content;
+
+	@Schema(description = "The list of participants who have read the message.")
```

Comment:
Not only participants. I think that the readBy-List can contain other persons than participants?